src/fides/api/ops/api/v1/endpoints/user_endpoints.py

```diff
@@ -252,7 +265,7 @@ def update_managed_systems(
             user.set_as_system_manager(db, system)
 
     # Removing systems for which the user in no longer a manager
-    for system in user.systems:
+    for system in user.systems.copy():
```

Comment:
is this just to be explicit that we're not modifying `user.systems`?

Reply:
Thanks for calling this out, I didn't explain this or add a test for it (but added one in the latest commit).  This is a bugfix.

Big picture, if you're looping through a list and potentially deleting elements in that list, you should copy the list first. This is because deleting an element in the list can change the index of subsequent elements, which can cause elements to get skipped that I need to delete.  I copy user.systems so I can modify them.

Reply:
oh I see, I didn't think of that. Thanks for clarifying!